Porting/how_to_write_a_perldelta.pod

```diff
@@ -89,35 +100,59 @@ For the start of a new stable branch, follows this formula:
     This document describes differences between the 5.10.0 release
     and the 5.12.0 release.
 
-Clearly this sets the scope of which changes are to be summarised in the rest
-of the document.
+This clearly sets the scope of which changes are to be summarised in the
```

Comment:
`s/summarised/summarized`:  `pod/perldocstyle.pod` says that we have standardized on "American spelling of words and expression of phrases."